src/lib/app-helper.js

```diff
@@ -51,8 +51,37 @@ async function runPackageScript (scriptName, dir, cmdArgs = []) {
   aioLogger.debug(`running npm run-script ${scriptName} in dir: ${dir}`)
   const pkg = await fs.readJSON(path.join(dir, 'package.json'))
   if (pkg && pkg.scripts && pkg.scripts[scriptName]) {
-    return execa('npm', ['run-script', scriptName].concat(cmdArgs), { cwd: dir, stdio: 'inherit' })
-  } else {
+
+    let termListeners = process.listeners('SIGINT')
+    process.removeAllListeners('SIGINT')
```

Comment:
why the side-effect of removing all listeners for the event? it's possible some script/code/hook might be listening as well for some reason